src/HelpersServiceProvider.php

```diff
@@ -19,10 +20,14 @@ public function boot()
         $router->prependMiddlewareToGroup('api', SecurityMiddleware::class);
 
         Validator::extend('unique_except_of_authorized_user', function ($attribute, $value) {
+            if (gettype($value) !== 'string') {
+                throw new UnprocessableEntityHttpException("The {$attribute} must be a string.");
+            }
```

Comment:
@Goodmain I've made small research how Laravel process cush case in `unique` rule

<img width="400" alt="image" src="https://github.com/RonasIT/laravel-helpers/assets/9486872/0a04bbeb-4f9d-4c1e-b818-75ceab0864a3">

<img width="984" alt="image" src="https://github.com/RonasIT/laravel-helpers/assets/9486872/c61ad0da-599f-4d22-aeb5-9c61efde0791">

<img width="979" alt="image" src="https://github.com/RonasIT/laravel-helpers/assets/9486872/7333e0da-40b1-4906-b953-af4ee3de7724">

It seems they made an Arr::flatten and take the first value as a arg.

Let's make the same logic, but let's take all values from the flutten array and use the whereIn clause in the query

Reply:
@DenTray Done